### PR TITLE
Workflow paths update for new RPMs

### PR DESCRIPTION
**Pull request by teo**

### CODE REVIEW

File: tasks/qc-daq-Dispatcher.yaml [hunk 1]
```diff
@@ -1,6 +1,10 @@
 name: qc-daq-Dispatcher
 defaults:
   log_task_output: none
+  _module_cmdline: >-
+    source /etc/profile.d/modules.sh && MODULEPATH={{ modulepath }} module load O2 QualityControl Control-OCCPlugin &&
+    {{ dpl_command }} | o2-qc
+  _plain_cmdline: "source /etc/profile.d/o2.sh && {{ dpl_command }} | o2-qc"
```
Comment (knopers8) [inline tasks/qc-daq-Dispatcher.yaml:7]: Stupid question, does `o2.sh` load also QualityControl?

**Comment by knopers8** [comment]
If this is proven to work, should I support both `_module_cmdline` and `_plain_cmdline` in the future auto-generated workflows or just the new way?

File: tasks/qc-daq-Dispatcher.yaml (hunk 1 above)
Comment (teo) [reply tasks/qc-daq-Dispatcher.yaml:7]: From what I understand, it prepares the env vars for all of our software. @awegrzyn can correct if I'm wrong.

**Comment by teo** [comment]
> If this is proven to work, should I support both `_module_cmdline` and `_plain_cmdline` in the future auto-generated workflows or just the new way?

Please support both, as this is the only way to be able to run both from new RPMs, old RPMs, and aliBuild output directory. Even when we drop support for old module-based RPMs, we'll still need this to support running from aliBuild output.

File: tasks/qc-daq-Dispatcher.yaml (hunk 1 above)
Comment (awegrzyn) [reply tasks/qc-daq-Dispatcher.yaml:7]: Yes, that's correct, it's a single file for all packages.

**Pull request merged**